src/mono/netcore/System.Private.CoreLib/src/System/Runtime/Loader/AssemblyLoadContext.Mono.cs

```diff
@@ -29,16 +29,16 @@ private static IntPtr InitializeAssemblyLoadContext(IntPtr thisHandlePtr, bool r
         private static extern void PrepareForAssemblyLoadContextRelease (IntPtr nativeAssemblyLoadContext, IntPtr assemblyLoadContextStrong);
 
         [System.Security.DynamicSecurityMethod] // Methods containing StackCrawlMark local var has to be marked DynamicSecurityMethod
-        private Assembly InternalLoadFromPath(string assemblyPath, string nativeImagePath)
+        private Assembly InternalLoadFromPath(string? assemblyPath, string? nativeImagePath)
         {
             StackCrawlMark stackMark = StackCrawlMark.LookForMyCaller;
 
-            assemblyPath = assemblyPath.Replace('\\', Path.DirectorySeparatorChar);
+            assemblyPath = assemblyPath?.Replace('\\', Path.DirectorySeparatorChar);
```

Comment:
I wasn't entirely sure what the right thing to do here was, since the implementation already doesn't handle nativeImagePath, so for now I just made it not null ref is assembly path is null.